src/client/visualizations/heat-map/heatmap-corner.scss

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015-2016 Imply Data, Inc.
+ * Copyright 2017-2019 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../imports';
+
+.heatmap-corner {
+  height: 100%;
+  background-color: #fff;
+  border: 1px solid $border-super-light;
+  border-left: 0;
+  border-top: 0;
+}
+
+.heatmap-corner-overflow-label {
+  @include ellipsis;
+  display: inline-block;
+}
+
+.heatmap-corner-row-title {
+  position: absolute;
+  bottom: 5px;
+}
+
+.heatmap-corner-column-title {
+  position: absolute;
+  top: 0;
+  right: 12px;
```

Comment:
I'm a bit afraid to ask where these magic numbers come from... Are they related to the font-size or some other elements of the UI?

Reply:
This is result of function of font size. Non trivial one.